app/styles/currency-input.less

```
@@ -0,0 +1,54 @@
+.currency-input-container {
+  position: relative;
+
+  .upf-infinite-select {
+    max-width: 100%;
+    .padding-xx-sm;
+
+    .upf-infinite-select__item {
+      padding: 0;
+      .padding-top-xxx-sm;
+      .padding-bottom-xxx-sm;
+    }
+
+    .row-selected span {
+      .text-color-default;
+    }
+  }
+
+  .symbol {
+    width: 22px;
+    font-weight: bold;
+  }
+
+  .currency-input {
+    .currency-selector {
+      border-right: 1px solid @border-color;
+      height: 100%;
+      display: flex;
+      align-items: center;
+      padding-right: 12px;
+      margin-right: 12px;
+    }
+
+    .phone-prefix {
+      white-space: nowrap;
+      .text-color-default-light;
+      .margin-right-xxx-sm;
+    }
+
+    input {
+      border: none;
+      background-color: transparent;
+    }
+
+    input:focus {
+      border: none;
+      outline: none;
+    }
+
+    &:focus-within {
+      border: 1px solid @upf-gray;
```

```suggestion
      border: 1px solid var(--color-border-default);
```
No good with this error